hazelcast/src/main/java/com/hazelcast/jet/impl/AbstractJobProxy.java

```diff
@@ -98,6 +99,7 @@
         this.container = container;
         this.lightJobCoordinator = isLightJob ? findLightJobCoordinator() : null;
         this.logger = loggingService().getLogger(Job.class);
+        logger.info("aaa coordinator for " + idToString(jobId) + " is " + lightJobCoordinator);
```

Comment:
probably leftover

Reply:
That was a test if you're paying attention - you passed 😉 
Removed.